certbot/tests/main_test.py

```diff
@@ -547,6 +547,26 @@ def test_plugins_no_args(self, _det, mock_disco):
 
     @mock.patch('certbot.main.plugins_disco')
     @mock.patch('certbot.main.cli.HelpfulArgumentParser.determine_help_topics')
+    def test_plugins_no_args_unprivileged(self, _det, mock_disco):
```

Comment:
Notes:

When `certbot plugins` is run as an unprivileged user:

* `certbot.util.set_up_core_dir` is called via `log.post_arg_parse_setup(config)` -> `setup_log_file_handler`.
* `certbot.util.set_up_core_dir` then throws `certbot.errors.Errors`.
*  `certbot.errors.Errors` is caught in `certbot.main` and is not raised when the `config.func` is `plugins_cmd`. Consequently `plugins_cmd` runs.

The test tries to simulate this by mocking `certbot.util.set_up_core_dir` and making it throw `certbot.errors.Errors.`